examples/python/pincell/build-xml.py

```diff
@@ -149,22 +149,18 @@
 # Register Cells with Universe
 root.add_cells([fuel, gap, clad, water])
 
-# Instantiate a Geometry and register the root Universe
+# Instantiate a Geometry and register the root Universe, and export to XML
```

Comment:
nbd, but perhaps this would read better as "...a Geometry, register the root Universe and export to XML" (i.e., add a comma before the first "and")